lib/Snippets.js

```diff
@@ -1,51 +1,50 @@
 var extractSnippets = require('./extractSnippets');
 var evaluateSnippets = require('./evaluateSnippets');
-var passError = require('passerror');
 
-function Snippets(snippets) {
-  this.items = snippets;
-}
+class Snippets {
+  constructor(snippets) {
+    this.items = snippets;
+  }
 
-Snippets.prototype.get = function(index) {
-  return this.items[index];
-};
+  evaluate(options, cb) {
+    return evaluateSnippets(this.items, options).then(() => this);
```

Comment:
I think we should avoid using `.then` when we have async/await available. Couldn't this be:

```suggestion
    await evaluateSnippets(this.items, options)
    return this;
```